README.md

```diff
@@ -18,9 +18,9 @@ running a single command.
 Current build status
 ====================
 
-Linux: [![Circle CI](https://circleci.com/gh/conda-forge/rever-feedstock.svg?style=shield)](https://circleci.com/gh/conda-forge/rever-feedstock)
-OSX: [![TravisCI](https://travis-ci.org/conda-forge/rever-feedstock.svg?branch=master)](https://travis-ci.org/conda-forge/rever-feedstock)
-Windows: [![AppVeyor](https://ci.appveyor.com/api/projects/status/github/conda-forge/rever-feedstock?svg=True)](https://ci.appveyor.com/project/conda-forge/rever-feedstock/branch/master)
+Linux: [![Circle CI](https://circleci.com/gh/conda-forge/feedstock-feedstock.svg?style=shield)](https://circleci.com/gh/conda-forge/feedstock-feedstock)
```

Comment:
This change happens when you rerender from a folder with a name `feedstock` instead of `package-name` or `package-name-feedstock`.

Reply:
Thanks for noticing @isuruf I'll put in a fix